pkg/option/config.go

```diff
@@ -969,6 +969,9 @@ const (
 	// HubbleSkipUnknownCGroupIDs specifies if events with unknown cgroup ids should be skipped
 	HubbleSkipUnknownCGroupIDs = "hubble-skip-unknown-cgroup-ids"
 
+	// HubbleMonitorEvents ...
```

Comment:
maybe something like this:
```suggestion
	// HubbleMonitorEvents specifies Cilium monitor events for Hubble to observe:
	// debug, trace-sock, drop-notify, trace-notify, policy-verdict-notify,
	// debug-capture, access-log, agent, lost. By default, Hubble observes all
	// monitor events.
```